pennylane/devices/task_qubit.py

```diff
@@ -0,0 +1,246 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This module contains a proxy qubit object for spawning multiple instances of 
+a given qubit type for run on a background scheduler.
+"""
+from typing import List, Union, Tuple, Dict
+from contextlib import nullcontext
+import pennylane as qml
+from pennylane import QubitDevice
+from .default_qubit import DefaultQubit
+from .._version import __version__
+
+try:
+    import dask
```

Comment:
Could the definitions starting from line 38 be defined after the try-except?

Reply:
Yep, good point :+1: